alf/algorithms/hypernetwork_algorithm.py

```diff
@@ -387,6 +430,11 @@ def train_iter(self, num_particles=None, state=None):
         if self._logging_training:
             if self._loss_type == 'classification':
                 logging.info("Avg acc: {}".format(acc))
+            if pinverse_loss > 0.:
+                if batch_idx == 0:
+                    batch_idx = 1
+                pinverse_loss = pinverse_loss / batch_idx
+                logging.info("Avg pinverse loss: {}".format(pinverse_loss))
```

Comment:
Is this for debugging purpose? If so, maybe move to summarize_train under if self._config.debug_summaries

Reply:
agreed